lib/event.rb

```diff
@@ -0,0 +1,38 @@
+class Event
+  attr_reader :name, :ages
+
+  def initialize(name, ages)
+    @name = name
+    @ages = ages
+  end
+
+  def max_age
+    p @ages.sort[-1]
+  end
+
+  def min_age
+    p @ages.sort[0]
+  end
+
+  def average_age
+    p (@ages.sum / @ages.length.to_f).round(2)
+  end
+
+  def standard_deviation_age
+    average_of_ages = (@ages.sum / @ages.length.to_f).round(2)
+
+    ages_less_average = []
+    @ages.each do |age|
+      ages_less_average << (age - average_of_ages).round(2)
```

Comment:
I would avoid doing the rounding until the very end. You may lose some precision if rounding too early.